command/pr_review.go

```diff
@@ -0,0 +1,137 @@
+package command
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/cli/cli/api"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	prCmd.AddCommand(prReviewCmd)
+
+	prReviewCmd.Flags().StringP("approve", "a", "", "Approve pull request")
+	prReviewCmd.Flags().StringP("request-changes", "r", "", "Request changes on a pull request")
+	prReviewCmd.Flags().StringP("comment", "c", "", "Comment on a pull request")
+
+	// this is required; without it pflag complains that you must pass a string to string flags.
+	prReviewCmd.Flags().Lookup("approve").NoOptDefVal = " "
+	prReviewCmd.Flags().Lookup("request-changes").NoOptDefVal = " "
+	prReviewCmd.Flags().Lookup("comment").NoOptDefVal = " "
+}
+
+var prReviewCmd = &cobra.Command{
+	Use:   "review",
+	Short: "TODO",
+	Args:  cobra.MaximumNArgs(1),
+	Long:  "TODO",
```

Comment:
Is documentation left to be addressed in follow-up?

Reply:
"yes" in that I forgot about it :) I can do it now and then augment to talk about the no args mode